core/src/test/scala/kafka/utils/ExitTest.scala

```diff
@@ -18,6 +18,7 @@
 package kafka.utils
 
 import java.io.IOException
+import org.apache.kafka.common.utils.Exit
```

Comment:
There's already a Java ExitTest which is identical, so this can be removed rather than migrated.

Reply:
agree to remove `ExitTest.scala`. 

unresolve this conversation as this comment is not addressed. @frankvicky Please update it in next comment, thanks!